api/endpoints/community.py

```diff
@@ -27,6 +29,14 @@ def make_blueprint(con):
         "disability_rate_2019":lambda: metric.disability_rate(year=2019, segment="all"),
         "belonging_rate_2017": lambda: metric.belonging(year=2017, segment="all"),
         "belonging_rate_2018": lambda: metric.belonging(year=2018, segment="all"),
+        "rent_burdened_2015": lambda: metric_rbu.rent_burdened(year=2015, segment="all"),
+        "rent_burdened_2016": lambda: metric_rbu.rent_burdened(year=2016, segment="all"),
+        "rent_burdened_2017": lambda: metric_rbu.rent_burdened(year=2017, segment="all"),
+        "rent_burdened_2018": lambda: metric_rbu.rent_burdened(year=2018, segment="all"),
+        "rent_burdened_2019": lambda: metric_rbu.rent_burdened(year=2019, segment="all"),
+        "rent_max": lambda: metric_rbu.rent_max_burdened(),
```

Comment:
The min and max metrics are missing a `GROUP BY area_number` in their query. Also you can remove the `period_end_year` from those two metrics.